src/main/java/com/inventoryservice/config/SecurityConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.inventoryservice.config;
+
+import com.inventoryservice.jwtutil.IsTokenValidFilter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+@Configuration
+@EnableWebSecurity
+@EnableMethodSecurity
+public class SecurityConfig {
+
+    @Autowired
+    private IsTokenValidFilter isTokenValidFilter;
+
+    public SecurityConfig(IsTokenValidFilter isTokenValidFilter) {
+        this.isTokenValidFilter = isTokenValidFilter;
+    }
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        return http.csrf().disable()
+                .authorizeHttpRequests()
+                .requestMatchers("/category/**").permitAll()
```

Comment:
Remove this and put on authorizehttprequest() method

Reply:
completed